src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -40,10 +43,13 @@ internal class StartHostAction : BaseAction
     {
         private const int DefaultPort = 7071;
         private const int DefaultTimeout = 20;
-        private const string Net6FrameworkDescriptionPrefix = ".NET 6.0";
+        private const string Net8FrameworkDescriptionPrefix = ".NET 8.0";
         private const string WindowsExecutableName = "func.exe";
         private const string LinuxExecutableName = "func";
         private const string InProc8DirectoryName = "in-proc8";
+        private const string InProc6DirectoryName = "in-proc6";
```

Comment:
Can we move this constants to a class scoping them to .NET (e.g., `DotnetConstants` or something along those lines)?